.github/workflows/daily-build-dev.yml

```
@@ -0,0 +1,41 @@
+      matrix:
+        node-version: [20.x]
+
+    steps:
+      - name: 'Setup ENV'
+        run: |
+          DATE=`date "+%Y%m%d"`
+          run_number=$(($GITHUB_RUN_NUMBER % 100))
+          run_number=$(printf "%02d" $run_number)
+          build_number="${DATE}${run_number}"
+          echo '$build_number='$build_number
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)